modelbaker/utils/db_utils.py

```diff
@@ -113,6 +112,7 @@ def get_configuration_from_sourceprovider(provider, configuration):
             and configuration.database
             and configuration.dbschema
         )
+    configuration.tool = mode
```

Comment:
Would this make things like

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/panel/tid_configurator_panel.py#L70

and

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/dataset_manager.py#L202

and

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/topping_wizard/ili2dbsettings_page.py#L274

and

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/topping_wizard/layers_page.py#L392

and

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/topping_wizard/models_page.py#L107

and

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/validate.py#L251

and

https://github.com/opengisch/QgisModelBaker/blob/fb9a349e44b74e8a12a17563a2e69573acdb2d1c/QgisModelBaker/gui/workflow_wizard/database_selection_page.py#L107

a bit redundant?

If so, it would be nice to remove them.

Reply:
Hm... yes I've overseen it. I don't really remember why it used to be like that. I'll check again.